nat-lab/tests/mac_wg.py

```diff
@@ -0,0 +1,19 @@
+import codecs
```

Comment:
maybe this can be moved to utils and used instead of python_wireguard for all platforms, since only `Key` is used from it

Reply:
I was thinking about that, but decided not to do this - I think that having real wireguard key generation being used on linux is valuable.